relay-server/src/actors/processor.rs

```diff
@@ -1505,6 +1511,15 @@ impl EnvelopeProcessorService {
         }
     }
 
+    fn handle_encode_metrics(&self, message: EncodeMetrics) {
+        #[cfg(feature = "processing")]
+        if let Some(ref store_forwarder) = self.inner.store_forwarder {
```

Comment:
Should we check explicitly for wether processing is enabled here or is this implicitly already the case through the store_forwarder being `Some`?

In any case I think this should still be an explicit check for the boolean.